docs/development-process/standard-release-process.md

```diff
@@ -0,0 +1,59 @@
+# Standard Release Process
+
+The following document is a description of the standard release process of
+Zarhus-related images. Precise steps and differences from the standard
+process will be described in layer-specific documentation.
+
+## Process steps
+
+1. Make sure that everything that should go into the given release is merged
```

Comment:
Tag should be pushed after PR is merged.

Reply:
Why? Tag is being assigned to the last commit of the PR, and will be merged into main with the commit. Or is it imposed by some CI?

Reply:
So the tag will be on main and not on possibly soon to be deleted branch.
Also pushing tag often starts release workflow / GitHub Actions (depends on configuration).

Reply:
Also tag doesn't change position during e.g. rebase. It would be much easier to forget to remove and then push tag again.

Reply:
> So the tag will be on main and not on possibly soon to be deleted branch.

Thought, that tag is bound only to a commit, and will be pushed from `develop` branch into `main` with the commit.

Reply:
If commit hash won't change then I guess yes. But it should still be done on main as most workflows rely on pushing tag to start release workflow.
And as I said tag doesn't change what it points to so it might get forgotten during rebase or changed after PR review. That's why I think it's better to push tag after PR is merged.

Reply:
Agreed, but steps 2,3, and 4 should be done before merging.

Reply:
Fixed 7141ab19b4a292911da04cfebea49c9139866ce7